runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/KafkaStreamingTest.java

```diff
@@ -53,32 +56,14 @@
  */
 public class KafkaStreamingTest {
   private static final EmbeddedKafkaCluster.EmbeddedZookeeper EMBEDDED_ZOOKEEPER =
-          new EmbeddedKafkaCluster.EmbeddedZookeeper();
+      new EmbeddedKafkaCluster.EmbeddedZookeeper();
```

Comment:
Once I have confidence that the runner's streaming support is clear enough (meaning a set of some ROS tests running) I'll consider.
I am however all for being "Kafka-aware" as this will probably be the main `UnboundedSource` users will use.